frontend/packages/dev-console/src/components/gitops/GitOpsListPage.tsx

```diff
@@ -38,12 +41,36 @@ const GitOpsListPage: React.FC = () => {
     };
   }, [baseURL, namespaces, nsError, nsLoaded]);
 
+  const [consoleLinks] = useK8sWatchResource({
+    isList: true,
+    kind: referenceForModel(ConsoleLinkModel),
+    optional: true,
+  });
+  let aLink;
+  _.filter(consoleLinks, (link: any) => {
+    if (link.spec.location === 'ApplicationMenu') {
+      if (link.metadata.name === "argocd") {
+        aLink = link;
+        return true;
+      }
+      return false;
+    }
+  });
+
   return (
     <>
       <Helmet>
         <title>Application Stages</title>
       </Helmet>
-      <PageHeading title="Application Stages" badge={<DevPreviewBadge />} />
+      <PageHeading 
+        title="Application Stages" 
+        badge={
+            <Split className="odc-gitops-list-page-heading">
+              {aLink && <ExternalLink href={aLink.spec.href} text="Argo CD" additionalClassName="odc-gitops-list-page-heading__argocd"/>}
+              <DevPreviewBadge />
+            </Split>
+        }>
+      </PageHeading> 
```

Comment:
Nit: This component is self-closing since it is empty.

Reply:
These issues are captured by the linter but I think you still havent added eslint to your code editor. Also when you compile the code you can see the lint errors in your terminal as well. The frontend job is also failing on this PR because of these lint errors.

Reply:
Thanks, I've updated this now but I do have both Prettier and ESLint installed and enabled so I'm not sure why it's not giving me errors for anything

Reply:
If you open your editor in root of console repository then it doesn't work correctly. You should open the editor in `console/frontend`. This could probably be a reason for the issue.